tests/support/identity.rs

```diff
@@ -51,6 +52,20 @@ where
 }
 
 impl Identity {
+    pub fn from_pem(s: &str) -> Arc<rustls::ClientConfig> {
+        use std::io::Cursor;
+
+        let mut roots = rustls::RootCertStore::empty();
+        roots
+            .add_pem_file(&mut Cursor::new(s))
+            .expect("add pem file");
+
+        let mut c = rustls::ClientConfig::new();
+        c.root_store = roots;
+        c.enable_tickets = false;
```

Comment:
We shouldn't need to worry about setting this in tests; it should be fine to leave out.